docs-chef-io/content/inspec/resources/aws_iam_user.md

```diff
@@ -121,7 +121,7 @@ it { should have_mfa_enabled }
 This will ensure the User has a console password set.
 
 ```ruby
-it { should have_console_password }
+it { should has_console_password }
```

Comment:
It should be have_console_password. has changes to have.
Please go through this link: https://rspec.info/documentation/3.9/rspec-expectations/RSpec/Matchers.html